docs/tools/fbsql/fbsql-kafka.md

```diff
@@ -0,0 +1,174 @@
+---
```

Comment:
I've written up a Jira with comments and suggestions:

https://molecula.atlassian.net/browse/DOCS-214